tools/introspection/include/iceoryx_introspection/introspection_app.hpp

```diff
@@ -130,7 +130,7 @@ class IntrospectionApp
     void printProcessIntrospectionData(const ProcessIntrospectionFieldTopic* processIntrospectionField);
 
     /// @brief prints table showing current mempool usage
-    void printMemPoolInfo(const MemPoolIntrospectionTopic& topic);
+    void printMemPoolInfo(const MemPoolIntrospectionInfo& introspectionInfo);
```

Comment:
Can probably be a const method, as can many other of those printXYZ methods. And also noexcept (we can always add this, but any exception thrown anyway will lead to a terminate.) Does not have to be done in the scope of this PR (since all other methods should be checked and potentially changed than as well)